<?xml version="1.0"?>
<launch>

  <arg name="map_frame"                default="map"/>
  <arg name="robot_frame"              default="base_footprint"/>
  <arg name="robot_radius"             default="0.35"/>
  <arg name="dt_radius"                default="0.2"/>
  <arg name="free_robot_pos_factor"    default="0.5"/>
  <arg name="map_scale"                default="1.0"/>
  <arg name="continuous_map_operation" default="true"/>

  <node name="rona_plan_node" pkg="rona_plan" type="rona_plan_node" output="screen">
    <param name="map_frame"                  type="string" value="$(arg map_frame)"                  />
    <param name="robot_frame"                type="string" value="$(arg robot_frame)"                />
    <param name="robot_radius"               type="double" value="$(arg robot_radius)"               />
    <param name="dt_radius"                  type="double" value="$(arg dt_radius)"                  />
    <param name="free_robot_pos_factor"      type="double" value="$(arg free_robot_pos_factor)"      />
    <param name="map_scale"                  type="double" value="$(arg map_scale)"                  />
    <param name="continuous_map_operation"   type="bool"   value="$(arg continuous_map_operation)"   />
  </node>
</launch>